docs/_docs/2_4_configurations.md

```diff
@@ -461,6 +487,119 @@ Property: `hoodie.compaction.daybased.target` <br/>
 Property: `hoodie.compaction.payload.class` <br/>
 <span style="color:grey">This needs to be same as class used during insert/upserts. Just like writing, compaction also uses the record payload class to merge records in the log against each other, merge again with the base file and produce the final record to be written after compaction.</span>
 
+### Bootstrap Configs
+Controls bootstrap related configs. If you want to bootstrap your data for the first time into hudi, this bootstrap operation will come in handy as you don't need to wait for entire data to be loaded into hudi to start leveraging hudi. 
+
+[withBootstrapConfig](#withBootstrapConfig) (HoodieBootstrapConfig) <br/>
+
+#### withBootstrapBasePath(basePath) {#withBootstrapBasePath}
+Property: `hoodie.bootstrap.base.path` <br/>
+<span style="color:grey"> Base path to bootstrap dataset into hudi. </span> 
+
+#### withBootstrapParallelism(parallelism = 1500) {#withBootstrapParallelism}
+Property: `hoodie.bootstrap.parallelism` <br/>
+<span style="color:grey"> Parallelism value to be used to bootstrap data into hudi </span>
+
+#### withBootstrapKeyGenClass(keyGenClass) (#withBootstrapKeyGenClass)
+Property: `hoodie.bootstrap.keygen.class` <br/>
```

Comment:
@bvaradar : I haven't filled in the desc for these configs. Can you help me with these.